pep-0384.txt

```diff
@@ -266,11 +267,9 @@ In addition, functions expecting ``FILE*`` are not part of
 the ABI, to avoid depending on a specific version of the
 Microsoft C runtime DLL on Windows.
 
-Module and type initalizer functions are not available
-(PyByteArray_Init, PyByteArray_Fini, PyBytes_Fini,
-PyCFunction_Fini, PyDict_Fini, PyFloat_ClearFreeList,
-PyFloat_Fini, PyFrame_Fini, PyList_Fini, PyMethod_Fini,
-PyOS_FiniInterrupts, PySet_Fini, PyTuple_Fini).
+Module and type initializer and finalizer functions are not available
+(PyByteArray_Init, PyOS_FiniInterrupts
+and all functions ending in _Fini or _ClearFreeList).
```

Comment:
What's the reason for omitting the explicit list? Did the C API grow additional functions matching `_Fini` or `_ClearFreeList`?

Reply:
> Did the C API grow additional functions matching `_Fini` or `_ClearFreeList`?

Not that I know, but I haven't checked either. That's why I made the change: an explicit list is more likely to become outdated (even it's still up-to-date now, maybe it won't be in the future).

Anyway, this is just bikeshedding so I can easily revert that if you prefer.